### PR TITLE
fix lack_flag_check.unknown_flag

### PR DESCRIPTION
**Pull request by XueqiangWei**
1. The behavior is different after run "-cpu 'model_name',+unknown,check"
   between RHEL7 and RHEL6
   (after qemu output "CPU feature unknown not found" qemu exit in RHEL7,
    but qemu not exit in RHEL6)
2. The behavior in RHEL6 is not the one we really want, but it's
   just an usability issue (as using invalid flag names is not supported),
   so developper didn't backport the fix to RHEL6.
3. in auto script, add check point:
   qemu output "CPU feature unknown not found"

Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID 1265491

**Comment by maxujun** [comment]
Acked-by: xuma@redhat.com
PASS smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.RHEL.6.8.x86_64.io-github-autotest-qemu.flag_check.lack_flag_check.unknown_flag

File: qemu/tests/flag_check.py [hunk 1]
```diff
@@ -139,80 +140,96 @@ def get_guest_cpuflags(vm_session):
...
-    vm.verify_alive()
-    timeout = float(params.get("login_timeout", 240))
-    session = vm.wait_for_login(timeout=timeout)
-
-    # Get qemu model
-    host_cpumodel = utils_misc.get_host_cpu_models()
-    if guest_cpumodel not in host_cpumodel:
-        qemu_model = host_cpumodel[0]
+
+    if params.get("start_vm") == "no" and "unknown,check" in extra_flags:
```
Comment (xutian) [inline qemu/tests/flag_check.py]: it hard to understand, please make it more clearly.

**Comment by MiriamDeng** [comment]
Test it pass according to current code.
smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u3.spice.qcow2.virtio_scsi.up.virtio_net.RHEL.6.8.x86_64.io-github-autotest-qemu.flag_check.lack_flag_check.unknown_flag
PASS

**Comment by xutian** [comment]
looks OK to me, merged thanks @XueqiangWei  @maxujun and @MiriamDeng

**Pull request merged**